_config.yml

```diff
@@ -21,7 +21,7 @@ description: "Refugees Emancipation Supporters Campaign #resp15"
 
 # This URL is the main adresse. Don't include a slash at the end.
 #
-url: ""
+url: "http://support.refugeesemancipation.com"
```

Comment:
I think, setting this URL here could cause broken links when deploying to staging or dev, don't you think?